### PR TITLE
Describe changes for 1.x.x released versions in changelog

### LINKED ISSUE #143: Describe changes for old released versions 1.x.x in changelog

### PR DESCRIPTION
**Pull request by psergee**
Closes #143

### CODE REVIEW

**Review by Totktonada — APPROVED** [review]
Thanks!

File: CHANGELOG.md [hunk 1]
```diff
@@ -68,17 +68,29 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
-### Added
-- Fixed eof detection
+### Fixed
+- Fixed eof detection.
 
 ## [1.0.2] - 2017-12-20
-### Added
-- Fixed request parsing with headers longer than 4096 bytes
+### Fixed
+- Fixed request parsing with headers longer than 4096 bytes.
```
Comment (ligurio) [inline CHANGELOG.md:96]: There were also: `1.0`, `1.0.1`, `0.0.1`:
Changes in 1.0: https://github.com/tarantool/http/compare/0.0.1...1.0
Changes in 1.0.1: https://github.com/tarantool/http/compare/1.0...1.0.1

I don't know is it worth to add them to changelog too or not.

File: CHANGELOG.md [hunk 2]
```diff
@@ -68,17 +68,29 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+### Added
+- Detach callback support for protocol upgrade implementations.
 
 ## [1.0.3] - 2018-06-29
-### Added
-- Fixed eof detection
+### Fixed
+- Fixed eof detection.
 
 ## [1.0.2] - 2017-12-20
```
Comment (ligurio) [inline CHANGELOG.md:94]: Looks correct.
Changes in 1.0.2: https://github.com/tarantool/http/compare/1.0.1...1.0.2

File: CHANGELOG.md [hunk 3]
```diff
@@ -68,17 +68,29 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
 
 ## [1.0.5] - 2018-09-03
+### Changed
+- Protocol upgrade: detaching mechanism is re-worked.
 
 ## [1.0.4] - 2018-08-31
+### Added
+- Detach callback support for protocol upgrade implementations.
 
 ## [1.0.3] - 2018-06-29
```
Comment (ligurio) [inline CHANGELOG.md:90]: Looks correct.
Changes in 1.0.3: https://github.com/tarantool/http/compare/1.0.2...1.0.3

File: CHANGELOG.md [hunk 4]
```diff
@@ -68,17 +68,29 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+
+### Fixed
+- Fixed buffer reading when timeout or disconnect occurs.
+- Fixed cookie formatting: stop url-encoding cookie path and quoting cookie expire date.
 
 ## [1.0.5] - 2018-09-03
+### Changed
+- Protocol upgrade: detaching mechanism is re-worked.
 
 ## [1.0.4] - 2018-08-31
```
Comment (ligurio) [inline CHANGELOG.md:86]: Looks correct.
Changes in 1.0.4: https://github.com/tarantool/http/compare/1.0.3...1.0.4

File: CHANGELOG.md [hunk 5]
```diff
@@ -68,17 +68,29 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
 
 ## [1.0.6] - 2019-05-19
+### Added
+- Custom logger per route support.
+
+### Fixed
+- Fixed buffer reading when timeout or disconnect occurs.
+- Fixed cookie formatting: stop url-encoding cookie path and quoting cookie expire date.
 
 ## [1.0.5] - 2018-09-03
```
Comment (ligurio) [inline CHANGELOG.md:82]: Looks correct.
Changes in 1.0.5: https://github.com/tarantool/http/compare/1.0.4...1.0.5

File: CHANGELOG.md [hunk 6]
```diff
@@ -68,17 +68,29 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Middleware support (for e.g. for centrally handling authorization).
 
 ## [1.1.0] - 2019-05-30
+### Added
+- Travis builds for tags.
 
 ## [1.0.6] - 2019-05-19
```
Comment (ligurio) [inline CHANGELOG.md:74]: In general, looks correct, but missed two user-visible commits in README:
- https://github.com/tarantool/http/commit/da7121cedf2614eebcd39cdc6a0c48ef75fde96b
- https://github.com/tarantool/http/commit/518d62c47fd1c7e414abe0ad86075fbbe174c95c
Changes in 1.0.6: https://github.com/tarantool/http/compare/1.0.5...1.0.6

File: CHANGELOG.md [hunk 7]
```diff
@@ -68,17 +68,29 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Middleware support (for e.g. for centrally handling authorization).
 
 ## [1.1.0] - 2019-05-30
```
Comment (ligurio) [inline CHANGELOG.md:70]: I would add a link to a full list of changes for every release section.
For example for 1.1.0:
```
## [1.1.0] - 2019-05-30

### Added

- Travis builds for tags.

[1.1.0]: https://github.com/tarantool/http/compare/1.0.6...1.1.0
```

Rendered markdown will convert 1.1.0 to a link to a full list with changes. I found it is convenient for everyone who will read changelog. It's my own opinion and feel free to ignore :)

**Review by ligurio — APPROVED** [review]
Looks fine, however, I left a number of non-blocking comments.

LGTM

**Review by 0x501D — APPROVED** [review]
LGTM

**Pull request merged**